login.py

```diff
@@ -0,0 +1,104 @@
+import requests
+import json
+import re
+from urllib import parse
+
+requests.packages.urllib3.disable_warnings(requests.packages.urllib3.exceptions.InsecureRequestWarning)
+
+OIDC_LOGIN_URL="http://localhost:5000/dp/v1/login"
+USERS_CONFIGURATION_PATH="oauth/users.json"
+CLIENTS_CONFIGURATION_PATH="oauth/clients-config.json"
+
+f = open(USERS_CONFIGURATION_PATH)
+data = json.load(f) 
+
+user_data = data[0]
+user_name = user_data['Username']
+password = user_data['Password']
+
+f = open(CLIENTS_CONFIGURATION_PATH)
+data = json.load(f)
+client_data = data[0]
+client_id = client_data['ClientId']
+client_secret = client_data['ClientSecrets'][0]
+
+
+# Calling http://localhost:5000/dp/v1/login
+session = requests.session()
```

Comment:
We're creating sessions, which *should* allow us to persist some cookies between requests, but then we're creating a new session for every request. Can we either use the session as it's intended, or drop it?